src/components/MeldDebug.tsx

```diff
@@ -3,13 +3,108 @@ import { Quad, Stream } from "@rdfjs/types";
 import { MeldReadState } from "@m-ld/m-ld";
 import N3 from "n3";
 import { useMeld } from "../hooks/useMeld";
+import { prettyPrintJson } from "pretty-print-json";
+
+import { query as runQuery } from "xql";
+
+import type { JsonValue } from "type-fest";
+import Head from "next/head";
 
-// TODO: Stop momentary double-display.
 export const MeldDebug = ({
   prefixes,
 }: {
   prefixes?: N3.WriterOptions["prefixes"];
-}) => {
+}) => (
+  <>
+    <Query />
+    <Turtle prefixes={prefixes} />
+  </>
+);
+
+const INITIAL_QUERY = `[
+  {
+    "@context": {
+      "icaltzd": "http://www.w3.org/2002/12/cal/icaltzd#"
+    },
+    "icaltzd:summary": "?",
+    "icaltzd:status": "?"
+  }
+]
+`;
```

Comment:
> The implementation of xQL still doesn't support lists

... but that doesn't affect this query because it picks out the Todos and not the `todoMVCList`